templates/quickstart-jira-dc-with-vpc.template.yaml

```diff
@@ -223,7 +223,7 @@ Parameters:
   # Jira DC template parameters
   CatalinaOpts:
     Default: ''
-    Description: Pass in any additional jvm options to tune Catalina
+    Description: Pass in any additional jvm options to tune Catalina.
```

Comment:
The dots are a new rule as well?

Reply:
Yes, it seems to be a new linter rule. W9006 Parameter must end in a full stop "."